compiler/modules/types/listProj.bal

```diff
@@ -27,8 +27,38 @@ function listProjBdd(Context cx, IntSubtype|true k, Bdd b, Conjunction? pos, Con
     }
 }
 
+public function listAllMemberTypes(Context cx, SemType t) returns [Range[], SemType[]] {
+    if t is UniformTypeBitSet {
+        return (t & LIST) != 0 ? [[{ min: 0, max: int:MAX_VALUE }], [TOP]] : [[{ min: 0, max: int:MAX_VALUE }], [NEVER]];
```

Comment:
I would like to extract variables positive int range with TOP and positive int range with NEVER, I just can't think of a name for them.